<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PreferencesCloud</class>
 <widget class="QWidget" name="PreferencesCloud">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>713</width>
    <height>558</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="cloudStorageGroupBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>129</height>
      </size>
     </property>
     <property name="title">
      <string>Subsurface cloud storage</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0">
       <widget class="QLabel" name="label_16b">
        <property name="toolTip">
         <string extracomment="Email address used for the Subsurface cloud storage infrastructure"/>
        </property>
        <property name="text">
         <string>Email address</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="label_16c">
        <property name="text">
         <string>Password</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="cloud_storage_pin_label">
        <property name="text">
         <string>Verification PIN</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QLabel" name="cloud_storage_new_passwd_label">
        <property name="text">
         <string>New password</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLineEdit" name="cloud_storage_email">
        <property name="toolTip">
         <string extracomment="Email address used for the Subsurface cloud storage infrastructure"/>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="cloud_storage_password">
        <property name="echoMode">
         <enum>QLineEdit::Password</enum>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QLineEdit" name="cloud_storage_pin">
        <property name="toolTip">
         <string extracomment="One time verification PIN for Subsurface cloud storage infrastructure"/>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QLineEdit" name="cloud_storage_new_passwd">
        <property name="echoMode">
         <enum>QLineEdit::Password</enum>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QCheckBox" name="save_password_local">
        <property name="text">
         <string>Save Password locally?</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="0">
    <layout class="QHBoxLayout" name="pwdResetLayout">
     <item>
      <widget class="QLabel" name="label_pwd_help">
       <property name="text">
        <string>If you have a cloud account and forgot your password, you can reset it here:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="resetPassword">
       <property name="text">
        <string>Reset Password</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_help1">
     <property name="toolTip">
      <string extracomment="Help info 1"/>
     </property>
     <property name="text">
      <string>To create a new cloud account:</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="label_help2">
     <property name="toolTip">
      <string extracomment="Help info 1"/>
     </property>
     <property name="text">
      <string>1) Enter an email address and a novel password that Subsurface will use to initialise the dive log in the cloud. Click Apply to send the above email address and password to the (remote) cloud server. Cloud storage email and password can only consist of letters, numbers, and '.','-','_', and '+'.</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="label_help3">
     <property name="toolTip">
      <string extracomment="Help info 1"/>
     </property>
     <property name="text">
      <string>2) The server responds by sending a verification PIN to the above email address (This is the only occasion that Subsurface uses the email address provided above). The above dialog now has a new PIN text box, not visible previously.</string>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QLabel" name="label_help4">
     <property name="toolTip">
      <string extracomment="Help info 1"/>
     </property>
     <property name="text">
      <string>3) Enter the PIN in the corresponding text box in the above dialog (this field is only visible while the server is waiting for email address confirmation). Click Apply again. The Subsurface cloud storage account will be marked as verified and the Subsurface cloud storage service is initialised for use.</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>

 <connections>

 </connections>

</ui>
